_posts/2020-05-22-changes-to-rules-policies.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Changes to our rules policies
+tags:
+  - rules
+  - policies
+---
+# Changes to our Rules Policies
+
+When ESLint was first created, the idea was to have a small set of 20-30 core rules that could get everyone started, and then to let the community build out its own rules to fill in any remaining gaps. We did, however, continue implementing core rules as it seemed helpful to the point where today there are 280 core rules. At any given time, more than half of open issues and pull requests are related to rules that take up a significant portion of the team's time.
+
+ESLint now benefits from a thriving ecosystem of rule-focused plugins like `eslint-plugin-node`, `eslint-plugin-import`, `eslint-plugin-react`, and many others. Tools like astexplorer.net and numerous tutorials have made it easier than ever for developers to write their own rules. The core team no longer needs to build up a critical mass of basic rules, so we can prioritize our time on long-awaited core features like parallel linting that will benefit the whole community. We're making some changes to how we prioritize changes to built-in rules, but as always, if a rule isn't quite right for your situation, you're encouraged to modify it as a custom rule for your own project and share it with the rest of the community. 
```

Comment:
```suggestion
ESLint now benefits from a thriving ecosystem of rule-focused plugins like `eslint-plugin-node`, `eslint-plugin-import`, `eslint-plugin-react`, and many others. Tools like [astexplorer.net](https://astexplorer.net)and numerous tutorials have made it easier than ever for developers to write their own rules. The core team no longer needs to build up a critical mass of basic rules, so we can prioritize our time on long-awaited core features like parallel linting that will benefit the whole community. We're making some changes to how we prioritize changes to built-in rules, but as always, if a rule isn't quite right for your situation, you're encouraged to modify it as a custom rule for your own project and share it with the rest of the community. 
```

Reply:
Missing a space between the link and the following "and"